### ISSUE TITLE
tspan argument should be checked to be a proper time interval

### ISSUE THREAD

**Issue by schillic** [JuliaReach/ReachabilityAnalysis.jl#732]
Passing `tspan=(1., 0.)` to `solve` should throw an error instead of computing a normal-looking but invalid result.

**Comment by mforets** [comment]
yes, the interval was meant to compute backwards approximation. 
iirc DiffEq has a similar behavior.
